app/assets/javascripts/IndividualResult.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+
+const [Number, String, Mixed, ObjectId] = [
+  mongoose.Schema.Types.Number,
+  mongoose.Schema.Types.String,
+  mongoose.Schema.Types.Mixed,
+  mongoose.Schema.Types.ObjectId,
+];
+
+const IndividualResultSchema = mongoose.Schema(
+  {
+    // Population Attributes
+    STRAT: Number,
+    IPP: Number,
+    DENOM: Number,
+    NUMER: Number,
+    NUMEX: Number,
+    DENEX: Number,
+    DENEXCEP: Number,
+    MSRPOPL: Number,
+    OBSERV: Number,
```

Comment:
This should be an Array. Double check the contents of that array.

Reply:
Could be its own Object that would contain the episode result (if there is one), the value, & the units